app/src/main/java/com/example/pokemobil/domain/base/BaseViewModel.kt

```diff
@@ -0,0 +1,34 @@
+package com.example.pokemobil.domain.base
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.example.pokemobil.domain.model.Resource
+import com.example.pokemobil.domain.model.Status
+import kotlinx.coroutines.launch
+
+open class BaseViewModel : ViewModel() {
```

Comment:
Package yapısı için dailyde konuşalım. Güzel olmuş ufak tefek düzeltmeler yaparız.